hi-ml-multimodal/src/health_multimodal/vlp/inference_engine.py

```diff
@@ -27,6 +27,24 @@ def __init__(self,
         self.image_inference_engine = image_inference_engine
         self.text_inference_engine = text_inference_engine
 
+    def get_similarity_score_from_raw_data(self,
+                                           image_path: Path,
+                                           query_text: str) -> float:
```

Comment:
I suggest supporting a sequence of strings as well, or even enforcing it. I think `get_embeddings_from_prompt` already supports that, so I think not much more needs to be changed here, except the call to `item`.

Reply:
Updated.

Reply:
We need a test with multiple prompts, I think you need to change the call to item etc.

Reply:
It's already added.